cmds/exp/ectool/ectool.go

```diff
@@ -34,12 +34,12 @@ func main() {
 		fmt.Fprintf(os.Stderr, "%v", err)
 	}
 	fmt.Printf("p %v\n", p)
-	if c, ok := chips[*defaultChip]; !ok {
+	c, ok := chips[*defaultChip]
```

Comment:
if you're going to do this, why not just
chip, ok := chips(...)
and avoid 
chip = c
below

Reply:
thx, make sense.